src/mame/drivers/gticlub.cpp

```diff
@@ -579,6 +583,26 @@ void gticlub_state::sound_memmap(address_map &map)
 
 /*****************************************************************************/
 
+WRITE16_MEMBER(gticlub_state::gn680_sysctrl)
+{
+	// bit 15 = watchdog toggle
```

Comment:
Empty write handlers are very bad for a number of reasons. 
At minimum I expect it to have a logerror, if the known bits bitbangs too much suppress it with an AND mask. 
Also it's a standard to postfix read/write handlers with an `_r` or a `_w` respectively.

Reply:
It was also like that in teraburst for hornet.cpp (which was a "problem" for as long as I can remember). I find it odd in teraburst's situation the game doesn't complain with a "GUN I/F ERROR".